exercises.cabal

```
@@ -15,14 +15,14 @@ copyright:           2021-2022 Haskell Beginners 2022 Course
 build-type:          Simple
 extra-doc-files:     README.md
                      CHANGELOG.md
-tested-with:         GHC == 8.10.7
+tested-with:         GHC == 9.2.4
 
 source-repository head
   type:                git
   location:            https://github.com/haskell-beginners-2022/exercises.git
 
 common common-options
-  build-depends:       base ^>= 4.14
+  build-depends:       base ^>= 4.16
```

This should be a wider constraint to test with both GHC 8.10  and GHC 9.2 (and GHC 9.0 as a consequence)

```suggestion
  build-depends:       base >= 4.14 && < 4.17
```

The `^>= 4.16` is syntax sugar for `>= 4.16 && < 4.17`

---

Thank you for the comment. I have update the versions and checked builds with both 8.10.7 and 9.2.4 versions.